algebra-laws/jvm/src/main/scala/algebra/laws/platform/Platform.scala

```diff
@@ -0,0 +1,10 @@
+package algebra.laws.platform
+
+private[laws] object Platform {
```

Comment:
since this is private, can we remove this and use:

https://github.com/typelevel/cats/blob/main/kernel-laws/jvm/src/main/scala/cats/platform/Platform.scala

Reply:
I would have liked to, but that one is also private! Since we don't share the same package. Ok to open the `cats-kernel` one up?